source/collections/Vector2D.ooc

```diff
@@ -38,6 +40,58 @@ Vector2D: class <T> {
 	_elementPosition: func (row, column: Int, columnCount := this columnCapacity) -> Int {
```

Comment:
Well, it depends on whether the count is the same as the capacity. In a `VectorList`, they are not the same - you can have capacity for elements you don't have. In this case, it looks like they are the same, in which case I agree `count` would be much less ambiguous.